integration_tests/utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
```

Comment:
One nit @AntoineRR ,

I am not a big fan of naming a file as `utils` as I believe that a `utils` file eventually becomes a dump of functions we don't understand where to put.

Can you name we name this file like `http_methods_helpers` or `http_methods_testing_helpers` ?

Reply:
Sure! I must say I got used to `utils` but I have no issue renaming the file